cmd/iam-store.go

```diff
@@ -1887,6 +1887,145 @@ func (store *IAMStoreSys) GetAllParentUsers() map[string]ParentUserInfo {
 	return res
 }
 
+// Assumes store is locked by caller. If users is empty, returns all user mappings.
+func (store *IAMStoreSys) listUserPolicyMappings(cache *iamCache, users []string) []madmin.UserPolicyEntities {
+	var r []madmin.UserPolicyEntities
+	usersSet := set.CreateStringSet(users...)
+	for user, mappedPolicy := range cache.iamUserPolicyMap {
+		if !usersSet.IsEmpty() && !usersSet.Contains(user) {
```

Comment:
I guess we can test 'if usersSet.IsEmpty()' once before for loop and return immediately ?

Reply:
Would this be correct? Passing no users means we want to return policies attached to all users, this would return nothing.